pkgs/stdenv/cross/default.nix

```diff
@@ -66,7 +66,7 @@ in lib.init bootStages ++ [
            else if crossSystem.isDarwin
              then buildPackages.llvmPackages.clang
            else if crossSystem.useLLVM or false
-             then buildPackages.llvmPackages_8.lldClang
```

Comment:
Who is actually using this cross `useLLVM`?

Reply:
WASM. I often use it when testing because its much faster to bootstrap.